docs/user/resources/04-30-30-gcp-nfs-backup-schedule.md

```diff
@@ -0,0 +1,71 @@
+# GcpNfsBackupSchedule Custom Resource
+
+The `gcpnfsbackupschedule.cloud-resources.kyma-project.io` custom resource (CR) represents the user-defined schedule for creating backup
+of the `GcpNfsVolume` instances at regular intervals. 
+- Creates the backups by creating the `gcpnfsvolumebackup.cloud-resources.kyma-project.io` resources on the specified interval.
+- Allows users to specify days and times in the form of CRON expressions to automatically create the backups. 
+- Automatically deletes the backups when the backup reaches the configured `maximum retention days`. 
+- Allows the users to temporarily `suspend` or `resume` the backup creation/deletion. 
```

Comment:
```suggestion
- Enables you to temporarily suspend or resume the backup creation/deletion. 
```

Reply:
If "suspend" and "resume" are UI elements, use bold font for them. Otherwise, no special formatting is necessary.